README.md

```diff
@@ -27,15 +27,15 @@
 
 </div>
 
-- [**Website**](#website)
-- [**Scope**](#scope)
-- [**Roadmap**](#roadmap)
-- [**Getting Started**](#getting-started)
-- [**Contributing**](#contributing)
-- [**Feedback**](#feedback)
-- [**Documentation**](#documentation)
-- [**License**](#license)
-- [**Trademark**](#trademark)
+- [Website](#website)
+- [Scope](#scope)
+- [Roadmap](#roadmap)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+- [Feedback](#feedback)
+- [Documentation](#documentation)
+- [License](#license)
+- [Trademark](#trademark)
```

Comment:
Auto-format doesn't like the bold links, fine to simplify it this way or should I revert?

Reply:
I think it's fine :+1: